cmd/version.go

```diff
@@ -15,6 +14,6 @@ var versionCmd = &cobra.Command{
 	Short: "Print the version number of Dunner",
 	Long:  `All software has versions. This is Dunners's`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("v0.1")
+		log.Println("v0.1")
```

Comment:
This makes the output to be `INFO[0000] v0.1` which is not nice, Shall we keep these as it is?

Reply:
Alright.